src/brsTypes/components/RoSGNode.ts

```diff
@@ -7,22 +7,59 @@ import { Int32 } from "../Int32";
 import { RoAssociativeArray } from "./RoAssociativeArray";
 import { RoArray } from "./RoArray";
 import { AAMember } from "./RoAssociativeArray";
+import { Float } from "../Float";
+import { Call } from "../../parser/Expression";
+import { objectExpression } from "@babel/types";
 
 class Field {
-    constructor(readonly type: string, readonly alwaysNotify: boolean) {}
+    private type: string;
+    private value: BrsType;
+    private observers = new Array<BrsString>();
+
+    constructor(value: BrsType, alwaysNotify?: boolean) {
```

Comment:
Should alwaysNotify be nullable at this level instead of handling it outside the field?  Maybe use a default constructor?  Looks like we're not using it yet, but it would be better if whatever eventually uses it doesn't need to do a null check every time.

Reply:
Good point!  We could set a default value with `constructor(value: BrsType, alwaysNotify: boolean = false)`